proposals/boost/boost-common/src/main/java/boost/common/config/ConfigConstants.java

```diff
@@ -100,12 +100,19 @@
     public static final String JDBC_43 = "jdbc-4.3";
     public static final String JPA_21 = "jpa-2.1";
     public static final String JPA_22 = "jpa-2.2";
+    public static final String CDI_12 = "cdi-1.2";
```

Comment:
Wondering if we should do more to separate out Liberty, TomEE, etc. - related strings...not that this PR started this.